src/odemis/dataio/test/catmaid_test.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Created on 19 Feb 2019
+
+@author: Thera Pals
+
+Copyright © 2019 Thera Pals, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+from __future__ import division
+
+import numpy
+import unittest
+from requests import ConnectionError
+
+from odemis import model
+from odemis.dataio import AuthenticationError
+from odemis.dataio.catmaid import open_data
+
+
+class TestCatmaid(unittest.TestCase):
+
+    def test_open_data(self):
+        """
+        Test requesting different tiles from different servers.
+        """
+        url = "catmaid://catmaid.neurodata.io/catmaid/"
+        acquisition = open_data(url)
+        acquisition = acquisition.content[0]
+        size = (1024, 1024)
+        tile = acquisition.getTile(0, 0, 8, depth=437)
+        self.assertEqual(tile.shape, size)
+        self.assertEqual(tile.max(), 246)
+        numpy.testing.assert_almost_equal(tile.metadata[model.MD_POS], [0.000262, -6.55e-05])
+        numpy.testing.assert_equal(tile.metadata[model.MD_PIXEL_SIZE], [1024e-9, 1024e-9])
+        tile = acquisition.getTile(7, 2, 4, depth=437)
+        self.assertEqual(tile.shape, size)
+        self.assertEqual(tile.max(), 255)
+        # test for a url with a specified project id and stack id
```

Comment:
Let's move this to a separate test case.
This way, if one of these servers go down, the other test case passes.